lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntime.cpp

```diff
@@ -608,6 +613,51 @@ bool SwiftLanguageRuntimeImpl::AddJitObjectFileToReflectionContext(
       });
 }
 
+bool SwiftLanguageRuntimeImpl::AddObjectFileToReflectionContext(
+    ModuleSP module, ObjectFile &obj_file, llvm::Triple::ObjectFormatType obj_format_type) {
+  auto obj_file_format = GetObjectFileFormat(obj_format_type);
+
+  auto &memory_reader = m_reflection_ctx->getReader();
+  auto *lldb_memory_reader = (LLDBMemoryReader *)&memory_reader; 
+  auto start_address = lldb_memory_reader->addModuleToAddressMap(module);
+
+  auto *section_list = obj_file.GetSectionList();
+  auto segment_iter = llvm::find_if(*section_list, [](auto segment) {
+    return segment->GetName() == "__TEXT";
```

Comment:
This is hardcoding details about MachO that should probably be behind some kind of abstraction.